### PR TITLE
Footer for desktop view

### PR DESCRIPTION
**Pull request by apanjain**

### CODE REVIEW

File: frontend/src/components/BlogCard/BlogCard.module.scss [hunk 1]
```diff
@@ -3,37 +3,35 @@
 .containersm {
     font-size: 17.125rem;
     width: 17.125rem;
-    max-width: 90%;
+    max-width: 90vw;
```
Comment (hacksdump) [inline frontend/src/components/BlogCard/BlogCard.module.scss]: Is the 'vw' intentional? Can % work?

File: frontend/src/components/Footer/DesktopView.module.scss [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    background: white;
+    left:0;
+    right:0;
+    margin: 0 auto;
+    position:relative;
+    display: flex;
+    flex-direction: column;
+    bottom:0;
+
+    font-size: 18px;
```
Comment (hacksdump) [inline frontend/src/components/Footer/DesktopView.module.scss]: Font sizes should preferably use the unit 'em'.

File: frontend/src/components/Footer/index.js [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
-                <Sitemap/>
-            </div>
-            <ScrollToTop/>
-            <Acknowledgement />
-        </div>
-    )
+class Footer extends Component {
+    constructor(){
+        super();
+        this.state ={
```
Comment (hacksdump) [inline frontend/src/components/Footer/index.js]: You also need to update state on resize.

```javascript
updateDimensions = () => {
    this.setState({ width: window.innerWidth, height: window.innerHeight });
};

componentDidMount() {
    window.addEventListener('resize', this.updateDimensions);
}

componentWillUnmount() {
    window.removeEventListener('resize', this.updateDimensions);
}
```

File: frontend/src/components/BlogCard/BlogCard.module.scss (hunk 1 above)
Comment (apanjain) [reply frontend/src/components/BlogCard/BlogCard.module.scss]: yes vw is required

File: frontend/src/components/Footer/index.js (hunk 3 above)
Comment (apanjain) [reply frontend/src/components/Footer/index.js]: Done

File: frontend/src/components/Footer/DesktopView.module.scss (hunk 2 above)
Comment (apanjain) [reply frontend/src/components/Footer/DesktopView.module.scss]: done

**Pull request merged**